File: Serene/Serene.Script/Imports/FormContexts/Meeting.MeetingAgendaForm.cs

namespace Serene.Meeting
{
    using Serenity;
    using Serenity.ComponentModel;
    using System;
    using System.Collections;
    using System.Collections.Generic;
    using System.ComponentModel;
    using System.Runtime.CompilerServices;
    using Serene.Northwind;

    public partial class MeetingAgendaForm : PrefixedContext
    {
        public MeetingAgendaForm(string idPrefix) : base(idPrefix) {}
    
        public IntegerEditor AgendaTypeId { get { return ById<IntegerEditor>("AgendaTypeId"); } }
        public IntegerEditor MeetingId { get { return ById<IntegerEditor>("MeetingId"); } }
        public IntegerEditor DisplayOrder { get { return ById<IntegerEditor>("DisplayOrder"); } }
        public StringEditor IsActive { get { return ById<StringEditor>("IsActive"); } }
        public IntegerEditor RequestedByContactId { get { return ById<IntegerEditor>("RequestedByContactId"); } }
        public StringEditor Tags { get { return ById<StringEditor>("Tags"); } }
        public StringEditor Title { get { return ById<StringEditor>("Title"); } }
        public StringEditor Description { get { return ById<StringEditor>("Description"); } }
        public IntegerEditor ImageFileId { get { return ById<IntegerEditor>("ImageFileId"); } }
    }
}

File: Serene/Serene.Web/Modules/Common/Imports/ServerTypings/Meeting.MeetingAgendaForm.ts
namespace Serene.Meeting {
    export class MeetingAgendaForm extends Serenity.PrefixedContext {
        static formKey = 'Meeting.MeetingAgenda';

    }

    export interface MeetingAgendaForm {
        MeetingId: Serenity.IntegerEditor;
        AgendaNumber: Serenity.IntegerEditor;
        Title: Serenity.StringEditor;
        Description: Serenity.StringEditor;
        AgendaTypeId: Serenity.IntegerEditor;
        RequestedByContactId: Serenity.IntegerEditor;
        Images: Serenity.StringEditor;
        Attachments: Serenity.StringEditor;
    }

    [['MeetingId', () => Serenity.IntegerEditor], ['AgendaNumber', () => Serenity.IntegerEditor], ['Title', () => Serenity.StringEditor], ['Description', () => Serenity.StringEditor], ['AgendaTypeId', () => Serenity.IntegerEditor], ['RequestedByContactId', () => Serenity.IntegerEditor], ['Images', () => Serenity.StringEditor], ['Attachments', () => Serenity.StringEditor]].forEach(x => Object.defineProperty(MeetingAgendaForm.prototype, <string>x[0], { get: function () { return this.w(x[0], (x[1] as any)()); }, enumerable: true, configurable: true }));
}

